docs/UserGuide.md

```diff
@@ -361,6 +427,10 @@ Action | Format, Examples
 **Edit** | `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`<br> e.g.,`edit 2 n/James Lee e/jameslee@example.com`
 **Find** | `find KEYWORD [MORE_KEYWORDS]`<br> e.g., `find James Jake`
 **List** | `list`
+**Add Tag** | `addtag n/TAGNAME` <br> e.g., `addtag n/Potential Clients`
+**Edit Tag** | `edittag f/OLD_TAGNAME t/NEW_TAGNAME` <br> e.g., `edittag f/New Clients t/Prospective Clients`
+**List Tags** | `listtags`
+**Delete Tag** | `deletetag n/TAGNAME` <br> e.g., `deletetag n/friends`
```

Comment:
This is incomplete, you left out the `findtag` command.

Reply:
Sorry! I have added it in/